RoktDemo.xcworkspace/xcuserdata/danial.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -3,4 +3,22 @@
    uuid = "F886A9A5-9837-4F02-92A8-944C528CF2F5"
    type = "0"
    version = "2.0">
+   <Breakpoints>
```

Comment:
I don't believe you want to add breakpoint =)

Reply:
Removed